app/src/main/java/org/simple/clinic/drugs/selectionv2/entry/CustomPrescriptionEntrySheetv2.kt

```diff
@@ -5,7 +5,7 @@ import android.content.Intent
 import android.os.Bundle
```

Comment:
`Change Text -> TextInputLayout for labels in CustomPrescriptionEntrySheet` -> `Use TextInputLayout for labels in  CustomPrescriptionEntrySheet`?